src/main/java/org/sagebionetworks/bridge/services/AccountWorkflowService.java

```diff
@@ -708,16 +710,29 @@ private boolean isRequestThrottled(ThrottleRequestType type, String userId) {
         // Generate key, which is in the form of channel-throttling:[type]:[userId].
         String cacheKey = "channel-throttling:" + type.toString().toLowerCase() + ":" + userId;
```

Comment:
If we use CacheProvider and CacheKey, we have a definitive list of all the keys under which we're writing values... I would prefer this but only because I'm pretty OCD.